src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -731,6 +733,16 @@ You can set the [aggregation window duration](/docs/alerts-applied-intelligence/
   Baseline alert condition thresholds don't support editing the aggregation window. They use the 1 minute default.
 </Callout>
 
+### Sliding window aggregation
+
+You can use [sliding windows](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/create-smoother-charts-sliding-windows) to create smoother charts. This is done by creating overlapping windows of data.
+
+Once enabled, set the slide by duration to control how much overlap time your aggregated windows have. 
```

Comment:
Potentially add that slide by must divide equally into and be less than aggregation window duration.

Reply:
Sounds good!